php/commands/shell.php

```diff
@@ -35,7 +35,7 @@ public function __invoke( $_, $assoc_args ) {
 		if ( '\\Psy\\Shell' == $class ) {
 			\Psy\Shell::debug();
 		} else {
-			$repl = new $class( 'wp> ' );
+			$repl = new $class(  "\nwp> " );
```

Comment:
Could probably add a flag to `\WP_CLI\REPL` and strip out the newline just the first time the prompt is printed. Not sure about Psy or Boris though. Happy to open another issue/PR for that if you think it's worth the tinker?